modules/azure/azurite/azurite.go

```diff
@@ -0,0 +1,133 @@
+package azurite
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/go-connections/nat"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	// BlobPort is the default port used by Azurite
+	BlobPort = "10000/tcp"
+	// QueuePort is the default port used by Azurite
+	QueuePort = "10001/tcp"
+	// TablePort is the default port used by Azurite
+	TablePort = "10002/tcp"
+
+	// defaultCredentials {
+	// AccountName is the default testing account name used by Azurite
+	AccountName string = "devstoreaccount1"
+
+	// AccountKey is the default testing account key used by Azurite
+	AccountKey string = "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="
+	// }
+)
+
+// AzuriteContainer represents the Azurite container type used in the module
+type AzuriteContainer struct {
+	testcontainers.Container
+	Settings options
```

Comment:
Keeping this public to avoid the breaking change. OTOH, if we want to do the BC now, I'm fine to move this settings to private.

Reply:
I think it's never used, so we can fairly remove it.